x-pack/plugins/observability_solution/synthetics/public/apps/synthetics/components/monitor_add_edit/form/submit.tsx

```diff
@@ -107,9 +105,9 @@ export const ActionBar = ({
           </NoPermissionsTooltip>
         </EuiFlexItem>
       </EuiFlexGroup>
-      {monitorPendingDeletion && (
+      {monitorPendingDeletion.length > 0 && (
         <DeleteMonitor
-          configId={monitorId}
+          configIds={monitorPendingDeletion}
```

Comment:
nit `monitorPendingDeletion` is an array right? Name is a bit confusing. I would expect it to be `monitorsPendingDeletion`